stograde/student/clone.py

```diff
@@ -14,11 +15,12 @@ def clone_student(student: str, base_url: str):
 def clone_url(url: str, into: str = None):
     if into:
         logging.info('cloning {} into {}'.format(url, into))
-        _, output, _ = run(['git', 'clone', '--quiet', url, into])
+        status, output, _ = run(['git', 'clone', '--quiet', url, into])
     else:
         logging.info('cloning {}'.format(url))
-        _, output, _ = run(['git', 'clone', '--quiet', url])
+        status, output, _ = run(['git', 'clone', '--quiet', url])
     logging.debug(output)
-    if 'Permission denied' in output:
-        print('Permission denied when cloning from {}. Aborting'.format(url))
+    if status is RunStatus.CALLED_PROCESS_ERROR and 'Permission denied' in output:
+        print('Permission denied when cloning from {}.\n'
+              'Make sure the SSH key for this terminal is registered with StoGit.'.format(url))
```

Comment:
```suggestion
              'Make sure the SSH key for this device is registered with StoGit.'.format(url))
```

Reply:
oh, also, these error messages should have `file=sys.stderr` as an argument.

Reply:
I used "terminal" because in the case of the Windows Subsystem for Linux, you can have multiple WSL instances, each which needs its own key.

Reply:
hmm. How about "Make sure that this SSH key is registered with StoGit."?

Reply:
That works